icons/phoenix/phoenix-original-wordmark.svg

```diff
@@ -1 +1 @@
008-.56.036-.558.036-.352.063-.353.062-.105.031-.106.031.04.275.04.274.009.012.01.012.555-.09.556-.09.706.019.705.02.206.12.206.12.132.197.133.197v1.617l-1.25.023-1.25.024-.234.103-.235.103-.193.186-.193.187-.086.291-.087.292v.898l.088.293.087.294.192.221.192.222.323.115.323.115.53-.03.529-.03.323-.096.323-.097.333-.167.332-.168.035.09.035.092v.284h.588v-4.8l-.088-.294-.087-.295-.217-.224-.217-.223-.242-.107-.243-.107-.438-.069zm15.284.003-.439.093-.439.094-.264.163-.264.164-.182.238-.182.239-.103.327-.103.327.018 1.639.018 1.639.195.395.194.395.247.166.246.166.177.07.176.07.264.056.265.056.647-.022.647-.023.46-.096.461-.096.136-.052.137-.052-.057-.252-.056-.251-.015-.014-.014-.015-.585.122-.585.12-.617.008-.618.006-.235-.08-.235-.079-.157-.11-.156-.11-.132-.271-.13-.271-.03-.698-.028-.699h3.779V63.24l-.067-.312-.066-.312-.138-.273-.137-.273-.223-.184-.222-.183-.321-.12-.32-.12-.474-.032zm-31.094.004-.87.003-.868.002-.498.795-.498.797-.04.025-.04.024-.52-.822-.52-.823H63.9l.136.22.135.22.803 1.284.802 1.285-.823 1.302-.824 1.303-.14.221-.141.222.895-.016.896-.016.503-.788.504-.788.055-.005.056-.006.51.793.51.794.896.016.896.017-.087-.134-.087-.134-.861-1.365-.862-1.364v-.114l.854-1.363.854-1.362.06-.111zm-8.293 0v6.056h1.764v-6.056h-.882zm20.751.003-.499.256-.5.257-.426.274-.426.275-.001-.338-.001-.337-.035-.133-.036-.132h-.574v6.055h.705v-4.383l.573-.358.573-.357.368-.186.367-.185-.003-.131-.003-.132-.041-.222zm39.816.01-.508.257-.507.256-.412.267-.412.267-.017-.468-.017-.467h-.641v6.055h.705l.002-2.19.003-2.19.57-.359.571-.36.368-.185.367-.185v-.336l-.036-.181zm-9.671.106-.34.017-.341.018-.618 2.748-.618 2.748-.034-.032-.034-.033-.797-2.715-.797-2.716-.478-.017-.478-.017-.054.194-.054.193-.755 2.571-.756 2.572-.04-.044-.04-.043-.605-2.72-.604-2.718h-.626v.131l.65 2.86.65 2.858.03.103.03.103h1.044l.055-.191.054-.191.718-2.498.717-2.5.048-.049.048-.05.082.286.082.284.703 2.454.703 2.454h.53l.53.001.67-2.954.67-2.954.027-.077zm15.284.006-.434.003-.433.003-1.189 1.493-1.188 1.493 1.235 1.531 1.235 1.532h.888l-.231-.28-.231-.278-.947-1.156-.946-1.156-.07-.114-.072-.113 1.135-1.406 1.135-1.405.056-.074zm-27.028.5.53.004.528.004.243.126.242.125.153.18.152.181.06.22.06.218.001.514v.515h-3.074l.033-.548.033-.548.097-.23.096-.232.154-.161.154-.16.27-.104zm14.345.058h1.354l.293.15.292.15.176.287.177.286v3.184l-.163.277-.163.278-.278.163-.278.162-.617.013-.617.012-.215-.06-.216-.06-.235-.203-.236-.201-.108-.28-.107-.278v-2.822l.079-.221.078-.222.15-.178.149-.177.242-.13zm-63.908.61h.588l.154.071.154.07.084.162.084.162v.594H50.27v-.535l.09-.173.088-.172.173-.09zm-5.47.089.165.122.166.121.103.202.103.201v2l-.103.2-.103.202-.15.11-.15.11-.32.028-.32.027-.235-.087-.236-.087-.101-.125-.103-.125-.064-.24-.065-.24.02-.915.02-.916.104-.19.103-.19.19-.102.19-.102.393-.002zm41.251 2.204-.016.753-.017.753-.358.19-.358.19-.345.09-.345.089-.443-.003-.444-.002-.16-.087-.16-.087-.071-.13-.072-.132-.04-.294-.041-.294.05-.321.052-.321.178-.179.178-.178 1.206-.018z"/></svg>
```

Comment:
Hmm, I wonder if we should use a vertical wordmark logo instead. It's really hard to see the logo if it's small.
I found a .webp vertical wordmark version of the log. Couldn't find a .svg version, and unable to share .wepb files on github it seems, but the logo can be found here: https://dribbble.com/shots/1612621-Phoenix-Framework-Logo

Reply:
Since is original art and just changes the conformation, I'm ok with the change.
I'll be doing that. 🙂